updog: change method of determining update availability (#993)
bottlerocket-os/bottlerocket: sources/updater/README.md

### Update wave
Updates may include "wave" information which provides a way for updates to be scheduled over time for groups of Bottlerocket hosts.

> Updog will find the update wave the host belongs to and calculate the time position allocated to the host given its seed.

Reviewer: ```suggestion
Updog will find the update wave the host belongs to and calculate its time position within the wave based on its `settings.updates.seed` value.
```
